specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/schemaregistry.json

```diff
@@ -1,132 +1,279 @@
         "x-ms-examples": {
-          "BatchAccountDelete": { "$ref": "./examples/OperationGroupGet.json" }
+          "Get schema ID": {
+            "$ref": "./examples/OperationSchema_GetIdByContent.json"
+          }
         },
-        "description": "This is a sample get operation, please see guidelines in azure-rest-api-specs repository for more info",
+        "operationId": "Schema_GetIdByContent",
+        "produces": [
+          "application/json"
+        ],
         "parameters": [
           {
-            "$ref": "#/parameters/SubscriptionIdParameter"
+            "name": "group-name",
+            "in": "path",
+            "description": "Schema group under which schema is registered.  Group's serialization type should match the serialization type specified in the request.",
+            "required": true,
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/ResourceGroupNameParameter"
+            "name": "schema-name",
+            "in": "path",
+            "description": "Name of the registered schema.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "schema-content",
+            "in": "body",
+            "description": "String representation of the registered schema.",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "serialization-type",
+            "in": "header",
+            "description": "Serialization type of the registered schema.  Must match serialization type of the specified schema group.",
+            "required": true,
+            "type": "string"
           }
         ],
         "responses": {
           "200": {
-            "description": "Describe the result of a successful operation.",
+            "description": "OK",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "URL location of schema, identified by schema group, schema name, and version."
+              },
+              "X-Serialization": {
+                "type": "string",
+                "description": "Serialization type for the schema being stored."
+              },
+              "X-Schema-Id": {
+                "type": "string",
+                "description": "References specific schema in registry namespace."
+              },
+              "X-Schema-Id-Location": {
+                "type": "string",
+                "description": "URL location of schema, identified by schema ID."
+              },
+              "X-Schema-Version": {
+                "type": "integer",
+                "description": "Version of the returned schema."
+              }
+            },
             "schema": {
-              "$ref": "#/definitions/Result"
+              "$ref": "#/definitions/SchemaId"
```

Comment:
Why does this return the schema id twice? It exists as both the response schema and the `X-Schema-Id` header. Another service question.

Reply:
Yeah this is just to make the headers consistent across the requests.

Reply:
Maybe this is an @hmlam question. But, the body of the response contains the schema ID, and the headers of the response also contain schema ID (via `X-Schema-Id`). Does it make sense to have the service return this data twice? I'm fine with it as long as it is intentional.